src/uniparc/adapters/uniParcConverter.ts

```diff
@@ -27,14 +27,11 @@ export type UniParcXRef = Partial<{
   lastUpdated: string;
   ncbiGi: string;
   organism: TaxonomyDatum;
-  // "properties" only there when database is "EMBL", and not for all of them.
-  // We shouldn't rely on that to display anything in the frontend, and it
-  // wasn't used in the previous website anyway
-  properties?: Array<{ key: string; value: string }>;
   proteinName: string;
   proteomeId: string;
   component: string;
-  version?: number; // might not always be there (e.g., for PRF xrefs)
+  chain: string;
+  version: number; // might not always be there (e.g., for PRF xrefs)
```

Comment:
I think I removed all optionnals within this object just because we already always wrapping the whole thing as a Partial. Could revert, but then most of the keys would be `?:` anyway.

Reply:
Ah I see. I didn't catch the use of Partial. And also the comment in the line below it is connected and it all makes sense now. So all good.